drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -53,23 +53,28 @@ struct mlx5_vdpa_caps {
 };
 
 struct virtq_info {
-	uint32_t                rqn;
-	void                    *rq_buf;
+	uint32_t                qn;
+	void                    *q_buf;
```

Comment:
we are going to remove the rq soon, but why this change?

Reply:
@shahafsh , this is what we discussed before the change was made